jenkins-node/mantid-builder/readme.md

```diff
@@ -1,19 +1,87 @@
 # Jenkins nodes for Mantid in Docker
 
-This describes how to deploy and managed a containerized build node.
+This describes how to deploy and manage a containerized build node.
 Such a node can perform any Linux based jobs.
 
-## Deployment
+The instructions are split by the OS used as the base:
+
+- [Cloud VM](#cloud-vm). Assumes a clean image deployed in a cloud that
+  is not yet set up to run Docker.
+- [RancherOS](#rancheros). Not recommended as this is no longer supported.
+
+## Cloud VM
+
+### OS deployment
+
+Use the wizard provided by the cloud provider, e.g. Openstack for SCD cloud at ISIS.
+We're running containers so the base operating system does not really matter.
+If in doubt choose the latest non-gui Ubuntu version. When selecting
+the resources for the machine we suggest atleast 8 cores, 32GB RAM and
```

Comment:
There needs to be a space between `at` and `least`.

Reply:
Thanks. I've fixed that and amended my last commit.